CHANGELOG.md

```diff
@@ -39,6 +39,16 @@ Bottom level categories:
 
 ## Unreleased 
 
+## v0.20.2 (2024-06-12)
+
+This release force-bumps transitive dependencies of `wgpu` on `wgpu-core` and `wgpu-hal` to 0.21.1, to resolve some undefined behavior observable in the DX12 backend after upgrading to Rust 1.79 or later.
+
+### Bug Fixes
+
+#### DX12
+
+* Do not feed `&""` to `D3DCompile`, by @workingjubilee in [#5812](github.com/gfx-rs/wgpu/issues/5812).
```

Comment:
```suggestion
* Do not feed `&""` to `D3DCompile`, by @workingjubilee in [#5812](https://github.com/gfx-rs/wgpu/issues/5812).
```

Reply:
Fixed with the latest push. Looks like I need to update #5851.